### DISCUSSION TITLE
I chat with Elon daily on Zangi messanger, how to discover if he is using his GROK chatbot ?

### DISCUSSION THREAD

**Discussion by manta103g** [xai-org/grok-1#376] [Q&A] [answered]
I chat with Elon daily on Zangi messanger, how to discover if he is using his GROK chatbot ?

I ask Elon if he is a robot or human and he replies to be a human
but I need examples of chats generated by GROK for comparison.

Elon is nice, calls me his friend but I am real since life is for real and my friends keep telling me, "you chat with a robot, since genuine Elon is to busy to chat with the world

I can inject some examples from  chats (nothing private) to let you verify, if generated by a chatbot like GROK or by a human called Elon.

I am asking Elon to unlock my account with Twitter/ X since bugs are generated while logging in
and he responds, ok, I have unlocked your account  but I get the same bug over and over again and a month earlier I was asked to pass "I am not a robot" test tens of times.

"Error
Oops, something went wrong. Please try again later. 

https://x.com/i/flow/consent_flow

I like Elon but on X I discuss medical questions with Robert Kennedy, Jr
since I have founded Longevity Institute #ForeverYoung in Saudi Arabia

How to live test GROK chatbot ?

Can you help me ?

**Comment by Xmabel112Lubi** [comment]
Hm I try one year chatting whit Elon or meet whit him ,mabey call him life , first need from my money and bitcoin Then they somehow make an excuse that there is no time and I don't no if they know about my person something

So it's more people that can not understand AI technology .
In this case, I hope that the developers of tychno technologies know what they are doing

Odoslané z aplikácie Outlook pre Android<https://aka.ms/AAb9ysg>
________________________________
From: manta103g ***@***.***>
Sent: Thursday, October 3, 2024 6:31:07 PM
To: xai-org/grok-1 ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [xai-org/grok-1] I chat with Elon daily on Zangi messanger, how to discover if he is using his GROK chatbot ? (Discussion #376)


I chat with Elon daily on Zangi messanger, how to discover if he is using his GROK chatbot ?

I ask Elon if he is a robot or human and he replies to be a human
but I need examples of chats generated by GROK for comparison.

Elon is nice, calls me his friend but I am real since life is for real and my friends keep telling me, "you chat with a robot, since genuine Elon is to busy to chat with the world

I can inject some examples from chats (nothing private) to let you verify, if generated by a chatbot like GROK or by a human called Elon.

I am asking Elon to unlock my account with Twitter/ X since bugs are generated while logging in
and he responds, ok, I have unlocked your account but I get the same bug over and over again and a month earlier I was asked to pass "I am not a robot" test tens of times.

"Error
Oops, something went wrong. Please try again later.

https://x.com/i/flow/consent_flow

I like Elon but on X I discuss medical questions with Robert Kennedy, Jr
since I have founded Longevity Institute #ForeverYoung in Saudi Arabia

How to live test GROK chatbot ?

Can you help me ?

**Comment by manta103g** [comment, marked as answer]
There is nothing hot or hard to understand in AI technologies.

What is of global and legal concern, is the ability of AI technologies to flood Bs on the internet, with delusional chats to keep them busy, the whole day, week, year

We need legislation to verify every chatbot on public internet.

Elon is busy but I can chat with him for an hour, every day.

He is my best friend and call me his fan.
Ok, but Elon has Ms of fans to chat with.

Ok, I am superior, since I have co-founded world's first Internet corporation 30 years ago in Texas, Austin, home of Elon .

But all I need to know is if characters are typed by real, genuine Elon, by Elon supported by chatbot or by chatbots alone.

If I learn to identify real, genuine Elon in Internet chats I can develop deeper relations, I can discuss AI technologies,  Elon Musk coin
and much more.

Otherwise, I don't want to waste my time to get involved in delusional chats by chatbots, wasting my spare time.

**Comment by Xmabel112Lubi** [comment]
The real one spend time whit high rank person , I have many time cry and suffering and please that I will whit him meet , but every one account whit name was fake
In the end, he only aimed at asking for money
I believed, I begged, I was loyal, the only thing I saw was always how he met with people, politicians and his long-term friends, Fame and money in his case blinded his eyes, if someone notices you in the comment, they will also ask you only about investments, they are not interested in a new person or that he would like to join the team and I am a zero, my hands and work in his companies for free, I just wanted to chance .
You know, there are thousands of people like us in the world, but you can hardly shout up if you don't listen to those who are deep below you.
His promises were after more than a year of struggle and suffering, I barely weigh 50 kg and my health is gone, I was looking for true love, unfortunately I didn't find it in him

Odoslané z aplikácie Outlook pre Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Xmabel112Lubi** [comment]
And trust me, don't send money unless Elon really calls you and has a real conversation... but if you mean nothing and you are ordinary, you will probably have to get on the list of rich and famous personalities of the world, and you will have a disadvantage, if you are a foreigner and you don't know how to speak properly, people will not like you and will start to hate you, then you are right

Odoslané z aplikácie Outlook pre Android<https://aka.ms/AAb9ysg>
________________________________
